src/propIs.js

```diff
@@ -0,0 +1,28 @@
+var _curry3 = require('./internal/_curry3');
+var is = require('./is');
+var propSatisfies = require('./propSatisfies');
+
+
+/**
+ * Returns `true` if the specified object property is of the given type;
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Type
+ * @sig Type -> String -> Object -> Boolean
+ * @param {Function} type
+ * @param {String} name
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.is
+ * @see R.propSatisfies
+ * @example
+ *
+ *      R.propIs('x', Number, {x: 1, y: 2});  //=> true
+ *      R.propIs('x', Number, {x: 'foo'});    //=> false
+ *      R.propIs('x', Number, {});            //=> false
+ */
+module.exports = _curry3(function propIs(type, name, obj) {
+  return propSatisfies(is(type), name, obj);
+});
```

Comment:
Actually, there's a lot to be said for that.  If we could just convince that recalcitrant so-and-so to give up his concerns about functions that don't report their arity correctly, then we could accomplish this in a pretty straightforward manner.  But there's a huge wrinkle.  How do we know how many parameters each intermediate function should consume?  If we can insist that it's just one, then this could be straightforward.

It dovetails a bit with that [gist][bu] created, although of course it's still for multiple functions, not just two.

And we could do away with that pesky `useWith` altogether!

  [bu]: https://gist.github.com/buzzdecafe/b00905f6487725173ff7